app/models/post.rb

```diff
@@ -1,7 +1,9 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
-  has_many :likes, foreign_key: :post_id
-  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id, dependent: :destroy
+  has_many :comments, foreign_key: :post_id, dependent: :destroy
+  # has_many :comments,
+  # has_many :likes,
 
```

Comment:
- [ ] Please remove the above-commented codes in order to keep your code base neat 😉 this is among the best coding practices.